kibbeh/src/ui/NotificationElement/GenericNotification.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from "react";
+import { LgSolidRocket } from "../../icons";
+
+export interface GenericNotificationProps {
+  notificationMsg: ReactNode;
+  time: string;
+  actionButton?: ReactNode;
+  icon?: ReactNode;
+}
+
+export const GenericNotification: React.FC<GenericNotificationProps> = ({
+  notificationMsg,
+  time,
+  actionButton,
+  icon,
+}) => {
+  return (
+    <div className="flex items-center" style={{ width: "400px" }}>
```

Comment:
Let them go full width, as they'll be inside a container.

Reply:
Done. Should I center align everything, or is this ok?